form/src/main/java/com/ritense/form/web/rest/impl/FormIoFormFileResource.java

```diff
@@ -28,6 +28,7 @@
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.servlet.view.RedirectView;
 
+@Deprecated(since = "10.7.0")
 public class FormIoFormFileResource implements FormFileResource {
```

Comment:
Isn't this still used for S3?

Reply:
Maybe. But I just followed the story on this one.